fastlane/Fastfile

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+default_platform(:ios)
+
+TEST_RUNTIME = 'iOS 15.0'
+TEST_DEVICE = 'iPhone 11'
+
+platform :ios do
+  desc 'Builds the project and runs tests'
+  lane :test do
+    device = create_simulator(TEST_RUNTIME, TEST_DEVICE)
+
+    run_tests(
+      workspace: 'Aztec.xcworkspace',
+      scheme: 'Aztec',
+      device: device.name,
+      prelaunch_simulator: true,
+      buildlog_path: File.join(__dir__, '.build', 'logs'),
+      derived_data_path: File.join(__dir__, '.build', 'derived-data'),
+      ensure_devices_found: true
+    )
+
+    run_tests(
+      workspace: 'Aztec.xcworkspace',
+      scheme: 'WordPressEditor',
+      device: device.name,
+      prelaunch_simulator: true,
+      buildlog_path: File.join(__dir__, '.build', 'logs'),
+      derived_data_path: File.join(__dir__, '.build', 'derived-data'),
+      ensure_devices_found: true
+    )
+
+    destroy_simulator(device)
+  end
+end
+
+def create_simulator(runtime, device)
```

Comment:
I was surprised that this wasn't already a `fastlane` action – might be worth moving this into the release toolkit in the future (in order to be able to guarantee that we run tests on the desired runtime).